src/app/components/Configurations/index.jsx

```diff
@@ -0,0 +1,96 @@
+import { Tabs, Divider, Button } from "antd";
+import React, { useState, useEffect } from "react";
+import { useHistory } from "react-router-dom";
+
+import accountSvc from "../../../common/services/account.svc";
+
+import Settings from "../../../common/lib/settings";
+import Allowances from "../../../common/lib/allowances";
+import ListData from "../ListData";
+
+import "./styles.scss";
+
+const { TabPane } = Tabs;
+
+const settingsStore = new Settings();
+const allowancesStore = new Allowances();
+
+const Configurations = () => {
+  const [accounts, setAccounts] = useState([]);
+  const [settings, setSettings] = useState({});
+  const [allowances, setAllowances] = useState({});
+  const [currentAccount, setCurrentAccount] = useState(null);
+
+  const history = useHistory();
+
+  useEffect(() => {
+    return load();
+  }, []);
+
+  const load = () => {
+    return Promise.all([
+      accountSvc.getAll(),
+      accountSvc.getCurrentAccount(),
+      settingsStore.load(),
+      allowancesStore.load(),
+    ]).then((responses) => {
+      setAccounts(responses[0]);
+      setCurrentAccount(responses[1]);
+      setSettings(settingsStore.settings);
+      setAllowances(allowancesStore.allowances);
+    });
+  };
+
+  const resetAllowances = () => {
+    return allowancesStore.reset();
+  };
+
+  const updateCurrentAccount = async (accountId) => {
+    await accountSvc.setCurrentAccount(accountId);
+    await load();
+  };
+
+  const removeAccount = async (accountId) => {
+    await accountSvc.remove(accountId);
+    await load();
+  };
+
+  const goToAddAccount = () => {
```

Comment:
could this be a [`<Link>`](https://reactrouter.com/web/api/Link) from react router? then we would not need to have a manual function and manually do a history.push? (this is also for several cases)

Reply:
agree